Cargo.toml

```diff
@@ -28,9 +28,10 @@ chrono = "0.4"
 nix = "0.26"
 cap-std = "1.0"
 thiserror = "1.0"
-libc = "0.2.138"
-oci-spec = "0.6"
+libc = "0.2.141"
+oci-spec = { version = "0.6.0", features = ["runtime"] }
 sha256 = "1.1"
+libcontainer = { git = "https://github.com/utam0k/youki", rev = "03eb78d13a6493b1184aa40fb96c73d882975956" }
```

Comment:
Since [this PR](https://github.com/containers/youki/pull/1777) is merged now maybe this can be containers/youki rather than your fork?